server/data/comments.js

```diff
@@ -0,0 +1,36 @@
+const { ObjectId } = require('mongodb');
+const moment = require('moment');
+const { getProjectById } = require('./projects');
+const { projects } = require('../config/mongoCollections');
+const { isValidStr, internalServerErr } = require('../utils');
+
+const createComment = async (commentParam, user) => {
+	let { comment, projectId } = commentParam;
+	comment = isValidStr(comment, 'Comment');
+
+	const projectCollection = await projects();
+	const projectFind = await getProjectById(projectId);
+
+	let commentDate = moment().format('MM/DD/YYYY');
```

Comment:
Let's use `new Date()` for timestamp because we might have to sort them with time also

Reply:
This is changed.